FERMCHOP/FERMCHOP-IOC-02App/Db/Makefile

```diff
@@ -0,0 +1,18 @@
+TOP=../..
```

Comment:
I'm not convinced this second IOC makes sense as no beamline at ISIS has multiple fermi choppers. 

Additionally I think this has some C code/SNL in the ioc dir which would need moving to the support module to get a second IOC to work I think.